rotkehlchen/tests/api/test_snapshots.py

```diff
@@ -329,13 +329,17 @@ def assert_csv_export_response(
         reader = csv.DictReader(csvfile)
         count = 0
         for row in reader:
-            assert len(row) == 5
+            assert len(row) == 6
             assert row['category'] in (
                 'asset',
                 'liability',
             )
             assert row['amount'] is not None
             assert row['asset'] is not None
+            assert row['asset_symbol'] in (
```

Comment:
That's not what I mean. To make it future proof you should take the asset from `'asset'` and make sure `'asset_symbol'` is its symbol.

Will merge since it's green, but please do this in a subsequent PR